client/src/components/CourseTerms.tsx

```diff
@@ -98,69 +99,71 @@ export const CourseTerms = ({ course, variant, query }: CourseTermsProps) => {
 
   return (
     <div className='mr-auto flex flex-wrap gap-x-2'>
-      {instructorGroups.map(([term, instructors], i) => {
-        const season = term.split(' ')[0].toLowerCase();
-        return (
-          <div className='relative' key={term}>
-            <div
-              className={twMerge(
-                'my-1.5 flex text-sm dark:bg-neutral-700',
-                variant === 'small' ? 'px-2 py-1' : 'max-w-fit px-2 py-1',
-                termColorMap[season],
-                expandedState[i] ? 'rounded-xl' : 'rounded-full'
-              )}
-            >
-              {instructors.length > 0 ? (
-                <div className='flex flex-col gap-y-1'>
-                  {(expandedState[i]
-                    ? instructors
-                    : instructors.slice(0, 1)
-                  ).map((ins) => (
-                    <Link
-                      key={ins.name}
-                      to={`/instructor/${encodeURIComponent(ins.name)}`}
-                    >
-                      <div className='flex items-center space-x-1.5 whitespace-nowrap'>
-                        <SeasonIcon term={term} variant={variant} />
-                        <div className='pr-1 font-medium dark:text-gray-200'>
-                          <Highlight
-                            text={ins.name}
-                            query={query || undefined}
-                          />
+      {instructorGroups
+        .sort((a, b) => compareTerms(a[0], b[0]))
```

Comment:
@39bytes i literally just added 1 line of sort here github is tripping with its change viewers